acceptable/__main__.py

```diff
@@ -16,8 +16,11 @@
 
 
 def main():
-    cli_args = parse_args()
-    sys.exit(cli_args.func(cli_args))
+    try:
+        cli_args = parse_args()
+        sys.exit(cli_args.func(cli_args))
+    except Exception as e:
```

Comment:
What's the intent here? I've been taught not to catch Exception, and all you're doing is barfing anyway while printing the error message (and eating potentially valuable traceback). I'm sure you know this and there's a reason for this catching :)

Reply:
Thanks!